qa/workunits/fs/quota/quota.sh

```diff
@@ -98,11 +98,11 @@ stat --printf="%n %s\n" ancestor/p1/p2/parent/p3/file1
 
 setfattr -n ceph.quota.max_bytes -v 0 .
 setfattr -n ceph.quota.max_bytes -v 1 .
-setfattr -n ceph.quota.max_bytes -v 9223372036854775807 .
-expect_false setfattr -n ceph.quota.max_bytes -v 9223372036854775808 .
+setfattr -n ceph.quota.max_bytes -v 8388608T .
+expect_false setfattr -n ceph.quota.max_bytes -v 8388609T .
 expect_false setfattr -n ceph.quota.max_bytes -v -1 .
-expect_false setfattr -n ceph.quota.max_bytes -v -9223372036854775808 .
-expect_false setfattr -n ceph.quota.max_bytes -v -9223372036854775809 .
+expect_false setfattr -n ceph.quota.max_bytes -v -8388609T .
+expect_false setfattr -n ceph.quota.max_bytes -v -8388610T .
```

Comment:
Test both the human readable and the large integer (to confirm both work as expected).

Reply:
I've kept the existing test for large integers intact and added another one for testing human readable. PTAL